grades/tasks_pytest_test.py

```diff
@@ -44,6 +44,7 @@ def test_generate_course_certificates():
     )
     FinalGradeFactory.create(course_run__course=non_fa_course, passed=True)
     FinalGradeFactory.create(course_run__course=course, passed=False)
+    FinalGradeFactory.create(course_run__course=course, passed=True, status='pending')
```

Comment:
I get why this is the only change necessary, but can you update the comment on #L75-L76 to explain?